schedule/migration/aarch64_regression_test_offline.yaml

```diff
@@ -0,0 +1,109 @@
+name:           aarch64_regression_test_offline.yaml
+description:    >
```

Comment:
I guess the intention is to keep the format as it is presented here. Thus i suggest `description:    |` instead of `description:    >`.  Check https://stackoverflow.com/a/21699210/1462096 for details

Reply:
Yes, thanks for the info. I will update it following this suggestion.